e2e/specs/site.go

```diff
@@ -5,6 +5,13 @@ import (
 	"github.com/gravitational/robotest/e2e/model/ui"
 	sitemodel "github.com/gravitational/robotest/e2e/model/ui/site"
 
+	libdefaults "github.com/gravitational/robotest/lib/defaults"
+	"github.com/gravitational/robotest/lib/wait"
+
```

Comment:
no need for whitespace for projects from the same repository - let's group them together.